sdk/pinocchio/src/sysvars/slot_hashes/test_std.rs

```diff
@@ -0,0 +1,113 @@
+//! Tests that rely on the `std` feature (host-only helpers, alloc, etc.).
+
+use super::*;
+use crate::{
+    account_info::{Account, AccountInfo},
+    pubkey::Pubkey,
+};
+use core::ptr;
+extern crate std;
+use super::test_utils::*;
+use std::io::Write;
+use std::vec::Vec;
+
+#[test]
+fn test_from_account_info_constructor() {
+    use std::eprintln;
+    eprintln!("DEBUG: Test starting");
+    std::io::stderr().flush().unwrap();
```

Comment:
I think common convention is that tests should not print anything by default.
Debugging leftovers?